### PR TITLE
Add link to mate docs in README

### PR DESCRIPTION
**Pull request by sschriner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,3 +276,6 @@ If you are using Manticore on an academic work, consider applying to the [Crytic
 ## Demo Video from ASE 2019
 [![Brief Manticore demo video](https://img.youtube.com/vi/o6pmBJZpKAc/1.jpg)](https://youtu.be/o6pmBJZpKAc)
 
+## Tool Integrations 
+
+- [MATE: Merged Analysis To prevent Exploits](https://github.com/GaloisInc/MATE)
```
Comment (ekilmer) [inline README.md:281]: MATE is a large system, we should call out which subdirectories specifically use Manticore. Let's also link to the docs page(s) here https://galoisinc.github.io/MATE/dwarfcore.html and https://galoisinc.github.io/MATE/mantiserve.html

**Review by ekilmer — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -276,3 +276,9 @@ If you are using Manticore on an academic work, consider applying to the [Crytic
...
+  * [Mantiserve:](https://galoisinc.github.io/MATE/mantiserve.html) REST API interaction with Manticore to start, kill, and check Manticore instance
+  * [Dwarfcore:](https://galoisinc.github.io/MATE/dwarfcore.html) Plugins and detectors for use within Mantiserve engine during exploration 
+  *
```
Comment (ekilmer) [inline README.md:284]:
```suggestion
```

**Review by ekilmer — APPROVED** [review]
LGTM! Thanks!

File: README.md [hunk 3]
```diff
@@ -276,3 +276,8 @@ If you are using Manticore on an academic work, consider applying to the [Crytic
 ## Demo Video from ASE 2019
 [![Brief Manticore demo video](https://img.youtube.com/vi/o6pmBJZpKAc/1.jpg)](https://youtu.be/o6pmBJZpKAc)
 
+## Tool Integrations 
+
+- [MATE: Merged Analysis To prevent Exploits](https://github.com/GaloisInc/MATE)
+  * [Mantiserve:](https://galoisinc.github.io/MATE/mantiserve.html) REST API interaction with Manticore to start, kill, and check Manticore instance
+  * [Dwarfcore:](https://galoisinc.github.io/MATE/dwarfcore.html) Plugins and detectors for use within Mantiserve engine during exploration
```
Comment (Boyan-MILANOV) [inline README.md:283]:
```suggestion
  * [Dwarfcore:](https://galoisinc.github.io/MATE/dwarfcore.html) Plugins and detectors for use within Mantiserve engine during exploration 
  * [Under-constrained symbolic execution](https://github.com/GaloisInc/MATE/blob/main/doc/under-constrained-manticore.rst) Interface for symbolically exploring single functions with Manticore
```

**Review by Boyan-MILANOV — CHANGES_REQUESTED** [review]
Added a suggestion for under-constrained manticore docs. Other than that LGTM!

**Review by Boyan-MILANOV — APPROVED** [review]

**Pull request merged**